dupcop

```diff
@@ -49,25 +49,25 @@ const getLongestCommonStartString = (str1, str2) => {
 
 const copyFolder = (relativePathFrom, relativePathTo, substitutions, rootPath) => {
   console.log('copyFolder invoked with ', relativePathFrom, relativePathTo, substitutions, rootPath);
-  const pathFrom = path.join(rootPath, relativePathFrom);
-  const pathTo = path.join(rootPath, relativePathTo);
+  const pathFrom = join(rootPath, relativePathFrom);
+  const pathTo = join(rootPath, relativePathTo);
   const dirEntries = fs.readdirSync(pathFrom);
 
-  for (let entry of dirEntries) {
-    const entryFullPathFrom = path.join(pathFrom, entry);
-    const relativePartFrom = path.join(relativePathFrom, entry);
+  for (const entry of dirEntries) {
+    const entryFullPathFrom = join(pathFrom, entry);
+    const relativePartFrom = join(relativePathFrom, entry);
 
     const entryStat = fs.statSync(entryFullPathFrom);
 
     const newEntryName = substitute(entry, substitutions);
-    const entryFullPathTo = path.join(pathTo, newEntryName);
-    const relativePartTo = path.join(relativePathTo, newEntryName);
+    const entryFullPathTo = join(pathTo, newEntryName);
+    const relativePartTo = join(relativePathTo, newEntryName);
     if (entryStat.isDirectory()) {
       console.log('creaing directory ', entryFullPathTo)
       fs.mkdirSync(entryFullPathTo);
       copyFolder(relativePartFrom, relativePartTo, substitutions, rootPath)
     } else {
-      const content = fs.readFileSync(entryFullPathFrom).toString();
+      const content = fs.readFileSync(entryFullPathFrom, 'utf-8');
```

Comment:
`fs.readFileSync` will return `string` if encoding is given. I would argue that it is safer.